x-pack/plugins/security_solution/public/resolver/view/panel.test.tsx

```diff
@@ -278,16 +314,37 @@ describe(`Resolver: when analyzing a tree with no ancestors and two children and
               simulator().map(() => simulator().testSubject('resolver:panel:event-detail').length)
             ).toYieldEqualTo(1);
           });
-          it('should allow all fields to be copied', async () => {
-            const copyableFields = await simulator().resolve('resolver:panel:copyable-field');
-
-            copyableFields?.map((copyableField) => {
-              copyableField.simulate('mouseenter');
-              simulator().testSubject('resolver:panel:clipboard').last().simulate('click');
-              expect(navigator.clipboard.writeText).toHaveBeenCalledWith(copyableField.text());
-              copyableField.simulate('mouseleave');
-            });
-          });
+          describe.each([['user.domain', 'user.domain']])(
```

Comment:
We can test each field by adding more data here. I might fixup the test data so that the labels and values aren' the same text.